modules/release-notes/ref-release-notes-deprecated-functionalities.adoc

```diff
@@ -4,6 +4,51 @@
 
 This section lists deprecated functionalities in {product} {product-version}.
 
+[id="deprecated-functionality-rhidp-5350"]
+== Deprecation of `podman` and `ImageStreams` approach for Helm chart mirroring
```

Comment:
```suggestion
== Deprecation of `podman` and `ImageStreams` approach for Helm chart mirroring on OCP
```

This procedure was (and still is) specific to OCP.

Reply:
Done